spec/javascripts/components/stories/history/history_changes/history_change_spec.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { shallow } from "enzyme";
+import HistoryChange from "../../../../../../app/assets/javascripts/components/stories/History/HistoryChanges/HistoryChange";
```

Comment:
We probably can do it like this here:

```suggestion
import HistoryChange from "components/stories/History/HistoryChanges/HistoryChange";
```